orbit/pkg/luks/luks_linux.go

```diff
@@ -123,11 +141,27 @@ func (lr *LuksRunner) getEscrowKey(ctx context.Context, devicePath string) ([]by
 		if err != nil {
 			return nil, nil, fmt.Errorf("Failed re-prompting for passphrase: %w", err)
 		}
+
+		if len(passphrase) == 0 {
+			log.Debug().Msg("Passphrase is empty, no password supplied, dialog was canceled, or timed out")
+			return nil, nil, nil
+		}
+
+		err = lr.notifier.ShowProgress(ctx, dialog.ProgressOptions{
```

Comment:
If they hit this part isn't the other progress windows (line 121) still being displayed?

Reply:
Ah, sorry, just reviewing the new zenity code.

Reply:
every new prompt info/entry/progress will kill any existing running prompt to ensure we only display 1 at a time